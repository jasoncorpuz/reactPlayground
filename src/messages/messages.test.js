import React from 'react';
import ReactDOM from 'react-dom';
import Messages from './Messages';
import renderer from 'react-test-renderer';


it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Messages />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders the UI as expected', () => {
    const tree = renderer
        .create(<Messages name='messages' unread={4}/>)
        .toJSON();
    expect(tree).toMatchSnapshot();
}); 

// renders the component to a JSON file
// compare the rendered component to a saved version of the component
