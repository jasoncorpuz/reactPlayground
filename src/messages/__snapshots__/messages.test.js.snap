// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div
  className="messages"
>
  messages
  <div
    className="unread_count"
  >
    4
  </div>
</div>
`;
